### ISSUE TITLE
Primitives expect ISA circuits instead of abstract circuits

### ISSUE THREAD

**Issue by jyu00** [Qiskit/documentation#794]
### Select all that apply

- [ ] typo
- [ ] code bug
- [X] out-of-date content
- [ ] broken link
- [ ] other

### Describe the fix.

Primitives will soon stop accepting abstract circuits. The content needs to be updated to start moving people away from passing abstract circuits. The impacted docs include

- [X] Update wording to say primitives only accept ISA circuits
- [ ] Update all code examples
     - [ ] https://docs.quantum.ibm.com/start/hello-world
     - [ ] https://docs.quantum.ibm.com/run/primitives-get-started (done via #789)
     - [ ] https://docs.quantum.ibm.com/run/primitives-examples (done via #789)
     - [ ] https://docs.quantum.ibm.com/run/configure-runtime-compilation
     - [ ] https://docs.quantum.ibm.com/run/configure-error-mitigation
     - [ ] https://docs.quantum.ibm.com/run/advanced-runtime-options (not strictly necessary as it doesn't show how `circuit` is created, but better to rename them to `isa_circuit`)
     - [ ] https://docs.quantum.ibm.com/run/estimate-job-run-time
- [ ] Tutorials

cc @blakejohnson

**Comment by blakejohnson** [comment]
Thanks for opening this, @jyu00

**Comment by beckykd** [comment]
According to @pandasa123 , the tutorials all use locally transpiled circuits already, so no changes are necessary unless you want them renamed isa_circuit.

**Comment by jyu00** [comment]
The circuits are transpiled in the tutorials, but some places the abstract circuits are passed to the primitives instead of the transpiled ones, for example

- https://learning.quantum.ibm.com/tutorial/quantum-approximate-optimization-algorithm#step-4-post-process-return-result-in-classical-format
- https://learning.quantum.ibm.com/tutorial/chsh-inequality#step-3-execute-using-qiskit-primitives

(I only checked the featured tutorials). Do I need to open a separate issue for the tutorials?

**Comment by pandasa123** [comment]
@jyu00, fixed!

**Comment by abbycross** [comment]
@jyu00 Yes, learning platform content is not in this repo. But it looks like @pandasa123 is on it :-)
